client/src/LoginPage.css

```diff
@@ -0,0 +1,137 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;400&display=swap');
+
+
+.login-container { 
+    display: flex; 
+    justify-content: center; 
+  
+}
+.login-box { 
+   
```

Comment:
NIT: in the future could we please just clear any empty spaces :) Nothing major but just for clean practices